tests/Sinch.Tests/e2e/Verification/VerificationStatusTests.cs

```diff
@@ -8,23 +8,24 @@ namespace Sinch.Tests.e2e.Verification
 {
     public class VerificationStatusTests : VerificationTestBase
     {
+        // mocked based on oas file
         private readonly ReportSmsVerificationResponse _smsVerificationReportResponse =
             new ReportSmsVerificationResponse()
             {
-                Id = "1234567890",
+                Id = "some_string_value",
                 Method = VerificationMethod.Sms,
                 Price = new PriceBase
                 {
                     VerificationPrice = new()
                     {
-                        CurrencyId = "USD",
-                        Amount = 0.0127
+                        CurrencyId = "some_string_value",
+                        Amount = 1.1
                     }
                 },
-                Reason = Reason.Fraud,
-                Reference = "12345",
-                Source = Source.Intercepted,
-                Status = VerificationStatus.Fail
+                Reason = new Reason("some_string_value"),
```

Comment:
Same comment as in PR#44, whichever comes first

Reply:
That's commented on #44